btr-web/btr-common-components/utils/canadaPostAddressApi/find-v2.10.ts

```diff
@@ -0,0 +1,63 @@
+const CANADA_POST_FIND_API_URL =
+  'https://ws1.postescanada-canadapost.ca/AddressComplete/Interactive/Find/v2.10/json3.ws'
+
+// https://www.canadapost-postescanada.ca/ac/support/api/addresscomplete-interactive-find/#parameters
+// API : Find (v2.10)
+export type CanadaPostApiFindParams = {
```

Comment:
Thoughts about putting this into the interfaces folder? I would also be okay leaving it here

Reply:
I did, but wanted to leave them here as they are integral part of this specific API.